### PR TITLE
moving zebra background to tableCell  #78

### PR DESCRIPTION
**Pull request by eric-martinez-sociomantic**
#78

- [x] Tested in Browser Stack IE.

### CODE REVIEW

**Comment by eric-martinez-sociomantic** [comment]
@conor-cafferkey-sociomantic corrected.

File: src/Table/index.jsx [hunk 1]
```diff
@@ -8,19 +8,22 @@ import Text                 from '../Text';
...
-                )
-            }
-        </TableRow>
-    ) );
+            <TableRow key = { i }>
+                {
+                    row.map( ( col, j ) =>
+                        // eslint-disable-next-line react/no-array-index-key
+                        <TableCell
+                            key       = { j }>
```
Comment (conor-cafferkey-sociomantic) [inline src/Table/index.jsx]: Can be written on one line:
```
<TableCell key = { j }>
```

File: src/Table/table.css [hunk 2]
```diff
@@ -4,13 +4,21 @@
 .default
 {
     font-variant-numeric:	lining-nums tabular-nums;
+
+    &:first-child
+    {
+        padding-left: 0;
+    }
```
Comment (conor-cafferkey-sociomantic) [inline src/Table/table.css]: This doesn’t belong here 😉

File: src/TableCell/tableCell.css [hunk 3]
```diff
@@ -2,8 +2,29 @@
...
+    	{
+        	padding-bottom:	-1rem;
+    	}
+
+        &:first-child
+        {
+            padding-left: 0;
+        }
+    }
+}
```
Comment (conor-cafferkey-sociomantic) [inline src/TableCell/tableCell.css]: I see that the above code block came from `tableRow.css`. But it makes no sense here. (Also, negative `padding` isn‘t valid CSS 😛 )

This stuff should be removed, except for the `&:first-child` bit which should move to the `.default` block below.

File: src/Table/index.jsx [hunk 4]
```diff
@@ -68,14 +71,15 @@ const Table = ( {
...
+                {
                     if ( typeof columns[ index ] === 'object' )
-                            {
+                    {
                         const title = columns[ index ].title;
                         const size  = columns[ index ].size;
 
                         return React.cloneElement( cell,
                             {
+                                className   : cssMap.cell,
```
Comment (conor-cafferkey-sociomantic) [inline src/Table/index.jsx]: There’s a small problem with this:

In theory, the cell might already have some custom `className`, that we’d like to preserve. For example you could set the `children` prop of the table to the following:
```
<TableRow className="customRowClass">
    <TableCell className="customCellClass1">abc</TableCell>
    <TableCell className="customCellClass2">123</TableCell>
    ...
</TableRow>
...
```

Setting `className` to `cssMap.cell` would override any custom `className`.

Check rows 91-92 below to see how custom `className` can be preserved.

**Comment by eric-martinez-sociomantic** [comment]
@conor-cafferkey-sociomantic  corrected.

**Pull request merged**